src/coverage-system/coverageservice.ts

```diff
@@ -133,9 +133,15 @@ export class CoverageService {
         if (this.configStore.manualCoverageFilePaths.length) { return; }
 
         const fileNames = this.configStore.coverageFileNames.toString();
+        let baseDir = this.configStore.coverageBaseDir;
+        if (workspace.workspaceFolders) {
+            const workspaceFolders = workspace.workspaceFolders.map((wf) => wf.uri.fsPath);
+            baseDir = `{${workspaceFolders}}/${baseDir}`;
+        }
+
         // Creates a BlobPattern for all coverage files.
         // EX: `**/{cov.xml, lcov.info}`
-        const blobPattern = `**/{${fileNames}}`;
+        const blobPattern = `${baseDir}/{${fileNames}}`;
```

Comment:
This will effect your other PR as well but I am guessing you have tested these together?
Although by my reading it should be mutually exclusive in that you can have either this blobPattern that has quite a long of flexibility or you can do the manual approach?

Reply:
1. Yes, they are mutually exclusive
1. Yes I have tested them together (and separately)
1. They will conflict in this file, after one of them is merged, so I'll have to update second review
1. This change technically gives enough flexibility for glob setup of any complexity. manual approach can still be used for simple cases where you have just one or two explicit lcov files you want to watch